src/client/components/hud/Initiative.tsx

```diff
@@ -89,6 +108,72 @@ export function InitiativeHUD() {
 
   const dispatch = useServerDispatch();
 
+  const mapObjectsRef = useServerStateRef(
+    (state) =>
+      state.maps.entities[myself.currentMap]?.objects ?? EMPTY_ENTITY_COLLECTION
+  );
+  const viewPortSizeRef = useContext(ViewPortSizeRefContext);
+  const setTargetTransform = useContext(SetTargetTransformContext);
+  const setSelection = useRecoilCallback(
+    ({ snapshot, set, reset }) =>
+      (characterIds: RRCharacterID[]) => {
+        const tokens = characterIds.flatMap(
+          (characterId) =>
+            entries(mapObjectsRef.current).find(
+              (o) => o.type === "token" && o.characterId === characterId
+            ) ?? []
+        );
+        const ids = tokens.map((t) => t.id);
+        const currentIds = snapshot
+          .getLoadable(selectedMapObjectIdsAtom)
+          .getValue();
+        if (
+          (tokens.length > 0 && ids.length !== currentIds.length) ||
+          ids.some((id) => !currentIds.includes(id))
+        ) {
+          if (canEdit) {
+            snapshot
+              .getLoadable(selectedMapObjectIdsAtom)
+              .getValue()
+              .forEach((id) => reset(selectedMapObjectsFamily(id)));
+            set(selectedMapObjectIdsAtom, ids);
+            ids.map((id) => set(selectedMapObjectsFamily(id), true));
+          }
+          const size = viewPortSizeRef.current;
+          setTargetTransform(
+            translate(
+              -tokens[0]!.position.x + size.x / 2,
+              -tokens[0]!.position.y + size.y / 2
+            )
+          );
+        }
+      },
+    [canEdit, mapObjectsRef, viewPortSizeRef, setTargetTransform]
+  );
+
+  const lastRowId = useRef<RRInitiativeTrackerEntryID | null>(null);
+
+  const focusTokenOnTurnStart = useRRSettings()[0].focusTokenOnTurnStart;
+
+  useEffect(() => {
+    if (
+      focusTokenOnTurnStart &&
+      myself.isGM &&
```

Comment:
Do we really only want this feature for GMs?

Reply:
oups, indeed. A left-over from before the preference was introduced. Fixed now.